no_mutations.md

```diff
@@ -11,20 +11,266 @@ kernelspec:
   name: python3
 ---
 
+```{currentmodule} tskit
+```
+
 (sec_tskit_no_mutations)=
 
-# _Do you really need mutations?_
-% remove underscores in title when tutorial is complete or near-complete
+# Do you really need mutations?
+
+In tree sequences, the genetic genealogy exists independently of the mutations that
+generate genetic variation. This tutorial aims to illustrate
+cases where the genealogy on its own may be sufficent; a number of
+analyses do not actually require genetic variation.
```

Comment:
```suggestion
In tree sequences, the genetic genealogy exists independently of the mutations that
generate genetic variation, and often we are primarily interested in genetic variation
because of what it can tell us about those genealogies. This tutorial aims to illustrate
when we can leave mutations and genetic variation aside and study the genealogies directly.
```

Reply:
Nice!